build

```diff
@@ -1,16 +1,25 @@
 #!/bin/sh
 # SPDX-License-Identifier: BSD-3-Clause
 
-while getopts cd:hr: OPTCHAR; do
+# defaults
+. /etc/os-release
+DEB_DISTRO="${DEB_DISTRO:-$VERSION_CODENAME}"
+DEB_ARCH="${DEB_ARCH:-$(dpkg --print-architecture)}"
```

Comment:
nit: we could avoid this line by moving the `--arch` into a `set`.

Reply:
Do you mean there is another way to have default values for command line options? Can you point me to some documentation on how that works? I rarely work extensively with bash scripts and I don't know how that works.

Reply:
We don't need a default, see my proposals.